incubator/redis-cache/values.yaml

```diff
@@ -0,0 +1,49 @@
+# Default values for redis-cache.
+# This is a YAML-formatted file.
+# Declare variables to be passed into your templates.
+
+##Statefulset related configurations ss.yaml
+replicaCount: 3
+redis:
+  image:
+    repository: redis
+    tag: 3.0-alpine
+    pullPolicy: IfNotPresent
+  resources:
+    limits:
+      memory: 128Mi
+    requests:
+      memory: 128Mi
+microSentinel:
+  image:
+    repository: dhilipkumars/redis-sentinel-k8s
+    tag: latest
```

Comment:
This in combination with `IfNotPresent` will make it difficult to update if there is a newer version of this image. Would it be possible to tag it?